docs/conf.py

```diff
@@ -44,7 +44,8 @@
     "sphinx.ext.linkcode",
     "sphinx.ext.extlinks",
     "sphinx.ext.mathjax",
-    "sphinx_gallery.gen_gallery"
+    "sphinx_gallery.gen_gallery",
```

Comment:
I am considering sorting these alphabetically

Reply:
I'm very much for that

Reply:
Done.